checker/jdk/nullness/src/javax/xml/parsers/SecuritySupport.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2004, 2006, Oracle and/or its affiliates. All rights reserved.
+ * ORACLE PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ */
+
+package javax.xml.parsers;
+
+import java.security.*;
+import java.net.*;
+import java.io.*;
+import java.util.*;
+
+import org.checkerframework.checker.nullness.qual.*;
+
+/**
+ * This class is duplicated for each JAXP subpackage so keep it in sync.
+ * It is package private and therefore is not exposed as part of the JAXP
+ * API.
+ *
+ * Security related methods that only work on J2SE 1.2 and newer.
+ */
+class SecuritySupport  {
+
+	@SuppressWarnings("nullness")
```

Comment:
Please don't use tabs in source files.  Instead, use spaces.  Tabs display differently depending on one's editor -- you can see here that it causes the `@SuppressWarnings` not to line up with the code below it.

Reply:
Will make changes and keep in mind for next requests.
Thanks.